app/views/past_foreign_secretaries/index.html.erb

```diff
@@ -10,7 +10,7 @@
   </div>
 </div>
 
-<%= render partial: "featured_past_foreign_secretaries", locals: { past_foreign_secretaries_image: @selection_of_profiles } %>
+<%= render partial: "shared/featured_historical_figures", locals: { historical_figures_with_image: @selection_of_profiles, heading: "Selection of profiles"} %>
```

Comment:
Can that go in a locale file?

Reply:
Ah, good spot!

Reply:
Actually, I'm going to save this for the [refactoring follow-up ticket](https://trello.com/c/47S36ZI7/506-refactor-past-foreign-secretaries-past-pms-and-past-chancellors-in-collections-to-share-as-much-as-possible) as part of the acceptance criteria for that is going to be none of the past foreign secs / chancellors have hard-coded text, so will lump it in with all the other foreign secs locale stuff